src/traccuracy/_tracking_graph.py

```diff
@@ -152,10 +152,11 @@ def __init__(
                 pixel value of the node in the segmentation. Defaults to
                 'segmentation_id'. Pass `None` if there is not a label
                 attribute on the graph.
-            location_keys (tuple, optional): The list of keys on each node in
-                graph that contains the spatial location of the node. Every
-                node must have a value stored at each of these keys.
-                Defaults to ('x', 'y').
+            location_key (str | tuple, optional): The key or list of keys on each node
+                in graph that contains the spatial location of the node. Every
+                node must have a value stored at each of these provided keys.
+                If a single string, it is assume that the location is stored as a list
```

Comment:
```suggestion
                If a single string, it is assumed that the location is stored as a list
```

Reply:
Does the location key, if a single str, have to point to a `list` specifically? Can it point to a tuple? Is `np.ndarray` ok? I think we should be specific here, otherwise say `iterable`? I don't love forcing it to strictly be a list, as `np.ndarray` is pretty common, at a minimum

Reply:
I was trying to annotate the return type of `get_location` to be consistent with your suggestion. I tried Iterable, but we call 'len' on it. I tried Collection, but we index into it. I am settling for a type annotation of `list | tuple | np.ndarray`, I guess? Or I can cast to list when I return from get_location, but that feels wasteful.

Reply:
>  I am settling for a type annotation of list | tuple | np.ndarray

I think that's fine! Explicit is better than implicit. Would be good to test that `np.ndarray` works ok but I don't think it's strictly necessary for this PR